lib/Controller/CallController.php

```diff
@@ -111,6 +114,53 @@ public function getPeersForCall(): DataResponse {
 		return new DataResponse($result);
 	}
 
+	/**
+	 * Download the list of current call participants
+	 *
+	 * Required capability: `download-call-participants`
+	 *
+	 * @param 'csv'|'pdf' $format Download format
+	 * @return DataDownloadResponse<Http::STATUS_OK, 'text/csv'|'application/pdf', array{}>|Response<Http::STATUS_BAD_REQUEST, array{}>
+	 *
+	 * 200: List of participants in the call downloaded in the requested format
+	 * 400: No call in progress
+	 */
+	#[PublicPage]
+	#[RequireModeratorParticipant]
+	public function downloadParticipantsForCall(string $format = 'csv'): DataDownloadResponse|Response {
+		$timeout = $this->timeFactory->getTime() - Session::SESSION_TIMEOUT;
+		$participants = $this->participantService->getParticipantsInCall($this->room, $timeout);
+
+		if (empty($participants)) {
+			return new Response(Http::STATUS_BAD_REQUEST);
+		}
+
+		if ($format !== 'csv' && $format !== 'pdf') {
+			// Unsupported format
+			return new Response(Http::STATUS_BAD_REQUEST);
+		}
+
+		if ($format !== 'csv') {
+			// FIXME Remove once pdf was implemented.
+			return new Response(Http::STATUS_BAD_REQUEST);
+		}
+
+		$output = fopen('php://memory', 'w');
+		fputcsv($output, [
+			'name',
+			'type',
+			'identifier',
+		]);
```

Comment:
Should we prettify the output? (headers with capital letters) 
![image](https://github.com/user-attachments/assets/754ae895-d549-4ede-95db-fe705458e89f)

Reply:
We can do that for the PDF version later. For the CSV I would keep it "ugly"

Reply:
as-is should be less confusing for later use (like import)